src/main/java/de/agilecoders/wicket/requirejs/AmdModuleHeaderItem.java

```diff
@@ -19,78 +19,77 @@
+		return new AmdModuleHeaderItem(reference, name);
+	}
 
-    /**
-     * The name to use as an AMD identifier
-     */
-    private final String name;
+	/**
+	 * The name to use as an AMD identifier
+	 */
+	private final String name;
 
-    /**
-     * The resource reference that contributes the AMD module
-     */
-    private final JavaScriptResourceReference reference;
+	/**
+	 * The resource reference that contributes the AMD module
+	 */
+	private final JavaScriptResourceReference reference;
 
-    /**
-     * Creates a new {@code AmdModuleHeaderItem}.
-     *
-     * @param reference resource reference pointing to the javascript resource
-     * @param name      name that will be used as AMD identifier
-     */
-    public AmdModuleHeaderItem(JavaScriptResourceReference reference, String name) {
-        super(null);
-        this.reference = Args.notNull(reference, "reference");
-        this.name = Args.notNull(name, "name");
-    }
+	/**
+	 * Creates a new {@code AmdModuleHeaderItem}.
+	 *
+	 * @param reference resource reference pointing to the javascript resource
+	 * @param name      name that will be used as AMD identifier
+	 */
+	public AmdModuleHeaderItem(JavaScriptResourceReference reference, String name) {
+		super(null);
+		this.reference = Args.notNull(reference, "reference");
+		this.name = Args.notNull(name, "name");
+	}
 
-    @Override
-    public Iterable<?> getRenderTokens() {
-        return Collections.singletonList("amd-javascript-" + getName());
-    }
+	@Override
+	public Iterable<?> getRenderTokens() {
+		return Collections.singletonList("amd-javascript-" + getName());
+	}
 
-    @Override
-    public void render(Response response) {
-        IRequireJsSettings settings = RequireJs.settings();
-        AmdModulesRegistry modulesRegistry = settings.getModulesRegistry();
-        modulesRegistry.register(getName(), getReference());
-    }
+	@Override
+	public void render(Response response) {   
+		IRequireJsSettings settings = RequireJs.settings();
+		AmdModulesRegistry modulesRegistry = settings.getModulesRegistry();
+		modulesRegistry.register(getName(), getReference());
+	}
 
-    /**
-     * @return the AMD identifier
-     */
-    public String getName() {
-        return name;
-    }
+	/**
+	 * @return the AMD identifier
+	 */
+	public String getName() {
+		return name;
+	}
 
-    @Override
-    public JavaScriptResourceReference getReference() {
-        return reference;
-    }
+	@Override
+	public JavaScriptResourceReference getReference() {
+		return reference;
+	}
 
-    @Override
-    public List<HeaderItem> getDependencies() {
-    	
-        List<HeaderItem> dependencies = new ArrayList<>();
-        dependencies.add(new RequireJsHeaderItem());
-        return dependencies;
-    }
+	@Override
+	public List<HeaderItem> getDependencies() {
+		List<HeaderItem> dependencies = new ArrayList<>();
+		dependencies.add(new RequireJsHeaderItem());
+		return dependencies;
+	}
```

Comment:
Please revert the changes for the whole class.
It is just re-formatted without any meaningful code changes.